Model params convenience (#424)
SAP/ai-sdk-js: .changeset/sharp-pianos-press.md

---
'@sap-ai-sdk/orchestration': minor
---

> [Improvement] Provide suggestions for known keys in `model_params` object in LLM Module Config by refining type definition.

Reviewer: ```suggestion
[Improvement] Refine the type definition of the `model_params` property in the `LlmModuleConfig` to also include known properties.
```
